### PR TITLE
add description and examples for `exclusiveMinimum` keyword

### PR DESCRIPTION
**Pull request by AgniveshChaubey**
This PR addresses #93

### CODE REVIEW

File: content/2020-12/validation/exclusiveMinimum.markdown [hunk 1]
```diff
@@ -18,3 +18,69 @@ related:
   - vocabulary: validation
     keyword: multipleOf
 ---
+
+The `exclusiveMinimum` keyword is used to set an exclusive lower limit on numeric instances. It specifies that the numeric value being validated must be strictly greater than (not equal to) the provided minimum value.
+
+* Applies only to number data types (integers and floats).
```
Comment (jviotti) [inline content/2020-12/validation/exclusiveMinimum.markdown:24]: JSON doesn't define "floats" per se (which are specific representation of real numbers) but "real numbers". Can we say real numbers instead in these two PRs?

File: content/2020-12/validation/exclusiveMinimum.markdown [hunk 2]
```diff
@@ -18,3 +18,69 @@ related:
...
+* Validation succeeds if the number is strictly greater than the specified `exclusiveMinimum`.
+
+## Examples
```
Comment (jviotti) [inline content/2020-12/validation/exclusiveMinimum.markdown:27]: Can we tweak one of the examples to show off setting `exclusiveMinimum` to a real number and how that validates against integers & reals?

File: content/2020-12/validation/exclusiveMinimum.markdown [hunk 3]
```diff
@@ -18,3 +18,69 @@ related:
...
+
+{{<instance-pass `An instance with numeric value greater than 10 is valid`>}}
+15
+{{</instance-pass>}}
+
+{{<instance-fail `An instance with numeric value less than or equal to 10 is invalid`>}}
+9.5
+{{</instance-fail>}}
+
+* _**Note:** Here, the `exclusiveMinimum` takes precedence, even though `minimum` is 5. Only numbers greater than 10 are valid._
```
Comment (jviotti) [inline content/2020-12/validation/exclusiveMinimum.markdown:86]: Great example!

**Review by jviotti — CHANGES_REQUESTED** [review]

File: content/2020-12/validation/exclusiveMinimum.markdown (hunk 2 above)
Comment (AgniveshChaubey) [reply content/2020-12/validation/exclusiveMinimum.markdown:27]: Sure! Let me tweak the second one.

**Review by jviotti — APPROVED** [review]

**Pull request merged**